README.md

```diff
@@ -184,7 +192,38 @@ When executing a method, it is often useful to pass some parameters. Here's how
 
 The reserved key `$params` allows to pass parameters to a method and `$return` is the way to specify what to do with the result.
 
-By using _aliases_, it is possible to execute a method several times with different parameters:
+Note: in the previous examples, we didn't have parameters, so we didn't need to specify the `$return` in our queries, it was implicit.
+
+```js
+{
+  movies: {
+    count: true;
+  }
+}
+```
+
+is the same as:
+
+```js
+{
+  movies: {
+    $return: {
+      count: true;
+    }
+  }
+}
+```
+
+### Aliases
+
+By using _aliases_, it is possible to execute a method several times with different parameters, avoiding conflict names inside the values returned by the method.
```

Comment:
"avoiding conflict names inside the values returned by the method": de quelle méthode parles-tu ? Je pense que ce serait plus clair d'écrire : "avoiding conflict names inside the current context"